Source/WebCore/en.lproj/Localizable.strings

```diff
@@ -1453,6 +1456,9 @@
 /* WKErrorMalformedCredential description */
 "This credential is malformed" = "This credential is malformed";
 
+/* Title when a PDF needs a password to be unlocked */
+"This document is password protected." = "This document is password protected.";
```

Comment:
"password protected" or "password-protected"?

Reply:
This matches preview.